packages/api-openfin-demo/package.json

```diff
@@ -1,22 +1,23 @@
 {
- "name": "ssf-desktop-api-openfin-demo",
```

Comment:
It's the indentation - it's updated from 1 space, to 2 spaces. Happened automatically when I ran `npm i`